OrrRock/OrrRock/Views/VideoCollectionView/VideoCollectionViewCustomCell.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  VideoCollectionViewCustomCell.swift
+//  OrrRock
+//
+//  Created by dohankim on 2022/10/20.
+//
+
+import UIKit
+import SnapKit
+
+class VideoCollectionViewCell : UICollectionViewCell{
```

Comment:
```suggestion
class VideoCollectionViewCell : UICollectionViewCell {
```

중괄호 앞에 한 칸의 여백이 필요해보입니다 !